test/steps/kogitoapp.go

```diff
@@ -37,8 +37,10 @@ func registerKogitoAppSteps(s *godog.Suite, data *Data) {
 	// Deploy steps
 	s.Step(`^Deploy quarkus example service "([^"]*)" with configuration:$`, data.deployQuarkusExampleServiceWithConfiguration)
 	s.Step(`^Deploy spring boot example service "([^"]*)" with configuration:$`, data.deploySpringBootExampleServiceWithConfiguration)
-	s.Step(`^Create service "([^"]*)"$`, data.createService)
-	s.Step(`^Create service "([^"]*)" with configuration:$`, data.createServiceWithConfiguration)
+	s.Step(`^Create quarkus service "([^"]*)"$`, data.createQuarkusService)
+	s.Step(`^Create quarkus service "([^"]*)" with configuration:$`, data.createQuarkusServiceWithConfiguration)
+	s.Step(`^Create spring boot service "([^"]*)"$`, data.createSpringBootService)
+	s.Step(`^Create spring boot service "([^"]*)" with configuration:$`, data.createQuarkusServiceWithConfiguration)
```

Comment:
We need to change the Spec.Runtime if Quarkus or Spring Boot. However, I'm not sure if having different steps for each is too much. We could either:
a) Get the runtime from the name (we're following the approach of using quarkus or springboot naming, but this could not work in all the cases... )
b) Maybe only having one step with configuration, but don't really like to have the config - runtime there, since it's a bit verbose.

What do you think?

Reply:
I find it more readable to have the runtime into the text.

Solution `a` could be an idea. Check https://github.com/kiegroup/kogito-cloud-operator/blob/f67468dbe663617f2db0f00c6fb81db099191ac4/test/steps/kogitoapp.go#L32
same as `enabled|disabled`, we could allow only `spring boot|quarkus`
wdyt ?

Reply:
I thought about it. The problem is that the v1alpha1.RuntimeType expects either quarkus or springboot strings (not "spring boot"). Perhaps we could remove the spaces before doing the:

```
v1alpha.RuntimeType(runtime)
```

Or maybe we could just use an if statement instead of the v1alpha.RuntimeType(xxx) method. 

What do you think?

Reply:
we could use `springboot` directly into the step. Still readable to me.
@sutaakar @MarianMacik wdyt ?

Reply:
Other steps use "spring boot", don't really like to have some steps with "springboot" and others with "spring boot". Waiting for your feedback.

Reply:
> we could use springboot directly into the step. Still readable to me.

Sounds good to me, we can unify it for all steps.

Reply:
I've updated the steps using the optional string as Tristan suggested. Let me know your thoughts about this line:

```
kogitoApp.Spec.Runtime = v1alpha1.RuntimeType(strings.ReplaceAll(runtimeType, " ", ""))
```

Reply:
Using springboot in all the steps sounds fair enough to me as well.

Reply:
`springboot` seems better to me, as we reflect exact syntax to be used in CR

Reply:
Done using optional string (quarkus|springboot) and updated the rest of steps from "spring boot" to "springboot".